workbench/_api/routes/models.py

```
@@ -103,353 +102,19 @@ async def get_models(
-):
-
-    try:
-        values_V, indices_V, new_token_ids = get_remote_generate(job_id, state)
-        data = process_generation_results(
-            values_V, indices_V, new_token_ids, req, state
-        )
+        lens_models = j_lens.get_available_lenses()
+        for model in models:
+            name = model.get("name", "")
+            model["has_jacobian"] = name.rsplit("/", 1)[-1] in lens_models
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)